controllers/storagecluster_controller.go

```diff
@@ -19,10 +19,22 @@ package controllers
 import (
 	"context"
 
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/runtime"
 	ctrl "sigs.k8s.io/controller-runtime"
+	"sigs.k8s.io/controller-runtime/pkg/builder"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/event"
 	"sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/predicate"
+
+	ocsv1 "github.com/openshift/ocs-operator/api/v1"
+	odfv1alpha1 "github.com/red-hat-data-services/odf-operator/api/v1alpha1"
+)
+
+const (
+	HasStorageSystem = "odf.openshift.io/hasStorageSystem"
```

Comment:
Annotations should not have capital letters, best I can tell. Also OpenShift has somewhat weird conventions on how they generally want annotations, and really really don't want booleans. Maybe change this to something like `storagesystem.odf.openshift.io/watched-by` and set it's value to the corresponding StorageSystem name.

Reply:
Done